src/main/java/views/blackjack/blackjack.ftl.html

```diff
@@ -123,6 +123,41 @@
      });
 
 });
+
+//Function to handle hit requests from the user
+//Checks if the field of the hit button is "Hit"
+//If true then we make a post request to get a new card
+//Grab this new card, string configure it, and append
+//the new card to the users hand
+$("#dealHit").click(function(){
+    val = "";
+    data = "";
+    data = $("div.dealHit").text;
+
+    if(data == "Hit"){
+        var newCard;
+        //Send game data to get card{amount}
+        $.ajax({
+            type: "POST",
+            url: "/val/" + val,
```

Comment:
We can call the route whatever we want. Jacob called it "bet" for his because its a pretty good description of what it is going to be call in the controller. I think "hit" would be a better url name but that's just my opinion